### PR TITLE
Extruder tab layout CURA-2763

### PR DESCRIPTION
**Pull request by jackha**

### CODE REVIEW

File: resources/qml/SidebarHeader.qml [hunk 1]
```diff
@@ -61,103 +61,123 @@ Column
...
+                        {
+                            anchors.verticalCenter: parent.verticalCenter
+                            anchors.left: swatch.visible ? swatch.right : parent.left
+                            anchors.leftMargin: swatch.visible ? UM.Theme.getSize("default_margin").width / 2 : UM.Theme.getSize("default_margin").width
+                            anchors.right: parent.right
+                            anchors.rightMargin: UM.Theme.getSize("default_margin").width / 2
+
+                            color: control.checked ? UM.Theme.getColor("tab_checked_text") :
+                                        control.pressed ? UM.Theme.getColor("tab_active_text") :
+                                        control.hovered ? UM.Theme.getColor("tab_hovered_text") : UM.Theme.getColor("tab_unchecked_text")
```
Comment (sedwards2009) [inline resources/qml/SidebarHeader.qml]: I see that this line has been copied over, but I still wouldn't mind if someone took the time to replace it with a clearer function.

**Comment by nickthetait** [comment]
Do you mind posting a screenshot of what it looks like now?

**Comment by fieldOfView** [comment]
See https://github.com/Ultimaker/Cura/pull/1226

**Comment by nickthetait** [comment]
doh :face_with_head_bandage: thanks!

**Pull request merged**